src/components/DataTable.js

```diff
@@ -83,7 +83,7 @@ export default class DataTable extends React.Component {
           <tbody>
             {values.map((val, i) => (
               <tr key={i}>
-                {columns.map((col, j) => <td key={j}>{wide ? '' : col + ':'} {val[col]}</td>)}
+                {columns.map((col, j) => <td key={j}><span>{wide ? '' : col + ':'}</span> {val[col]}</td>)}
```

Comment:
Ideally the span would only be on the narrow view but I don't know how to do that. Any suggestions to achieve the same result are welcome.